docker/data/logstash/env2yaml/env2yaml.go

```diff
@@ -17,11 +17,12 @@ package main
 import (
 	"errors"
 	"fmt"
-	"gopkg.in/yaml.v2"
```

Comment:
Why was this moved? Doing the imports in alphabetical order seems like a pattern we want to keep.